x-pack/plugins/ml/public/application/services/anomaly_explorer_charts_service.ts

```diff
@@ -110,7 +110,7 @@ export class AnomalyExplorerChartsService {
     severity = 0,
     maxSeries?: number
   ): Observable<ExplorerChartsData> {
-    const bounds = this.timeFilter.getActiveBounds();
+    const bounds = this.getTimeBounds();
```

Comment:
The anomaly charts embeddable that is attached to the case is not using the time range I have set on the Anomaly Explorer in my tests. The time range set in the Anomaly Explorer should be preserved in the embeddable in the case.

Reply:
Fixed in https://github.com/elastic/kibana/pull/139628/commits/416bb9a58c4fecab5769baa77204a8e9923c3560